File: src/main/resources/templates/components/Controller.java
package {{packageName}};

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

import javax.servlet.http.HttpServletResponse;
import java.io.IOException;

@RestController
public class {{componentPrefix}}Controller {
    @GetMapping("/")
    public void redirect(HttpServletResponse response) throws IOException {
        response.sendRedirect("/demo");
    }

    @GetMapping(path = "/demo")
    public String demo() {
        return "This is a demo";
    }
}

File: src/main/resources/templates/components/Controller.kt
package {{packageName}}

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping(path = arrayOf("/demo"))
class {{componentPrefix}}Controller {
    @GetMapping
    fun demo(): String {
        return "This is a demo";
    }
}
